src/sass/patterns/_form-elements.scss

```diff
@@ -13,33 +13,16 @@
   display: grid;
   height: 100%;
 
-  .slds-accordion__summary {
-    @include icon-checkbox-layout;
+  .slds-accordion__section {
+    padding: $spacing-x-small;
   }
 
-  // using slds class since we can't access inner checkbox + label markup
-  .slds-checkbox__label {
-    &:hover {
-      .slds-form-element__label {
-        color: $brand-text-link-active;
-      }
-
-      .slds-checkbox_faux,
-      .slds-checkbox--faux {
-        border-color: $brand-text-link-active;
-      }
-    }
+  .slds-accordion__summary {
+    @include icon-checkbox-layout;
   }
 
-  // adding input focus since keyboard nav doesn't focus on spans
-  [type='checkbox'] {
-    &:focus {
-      + .slds-checkbox__label {
-        .slds-form-element__label {
-          color: $brand-text-link-active;
-        }
-      }
-    }
```

Comment:
Removed these just to revive the discussion from https://github.com/SFDO-Tooling/MetaShare/pull/149#discussion_r332730956 😄